### ISSUE TITLE
Last chunk not processed when batching

### ISSUE THREAD

**Issue by mactanner** [jtwebman/csv-batch#6]
First of all: thanks for the library! ;-)

I think there's a problem if the number of rows in the csv doesn't match the batch size exactly.
The records of the last chunk won't be processed. 

This test is currently failing:

```  
it('works if batch size does not match row count exactly', () => {
    const csv = `a,b,c\n1,2,3\n4,5,6\n7,8,9`;
    let callCount = 0;
    return csvBatch(createStreamFromString(csv), {
      batch: true,
      batchSize: 2,
      batchExecution: batch => {
        callCount++;
        return `processed batch ${callCount} size ${batch.length}`;
      }
    }).then(results => {
      assert.equal(callCount, 2);
      assert.equal(results.totalRecords, 3);
      assert.deepEqual(results.data, ['processed batch 1 size 2', 'processed batch 2 size 1']);
      assert.isEmpty(results.errors);
    });
  });
```

I've prepared a pull request, but currently I'm not allowed to submit it...

**Comment by jtwebman** [comment]
Nice find @mactanner. I need to add a contribution.md file. You should be able to fork the repo and open a pull request from your fork to this repo.

**Comment by jtwebman** [comment]
This was fixed with v2.0.1 release
